Code/Framework/AzFramework/Platform/iOS/AzFramework/Process/ProcessWatcher_iOS.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <AzFramework/Process/ProcessWatcher.h>
 #include <AzFramework/Process/ProcessCommunicator.h>
+#include <AzFramework/StringFunc/StringFunc.h>
```

Comment:
repeat: Can you change this to include StringFunc from AzCore instead?

```suggestion
#include <AzCore/StringFunc/StringFunc.h>
```

Reply:
> 

Also does the `ProcessWatcher_iOS.cpp` work on iOS?

Reply:
Not certain, but there is code here, so it needs this basic implementation to prevent missing symbols